src/libraries/NetCoreAppLibrary.props

```diff
@@ -60,6 +60,7 @@
       System.Drawing.Primitives;
       System.Dynamic.Runtime;
       System.Formats.Asn1;
+      System.Formats.Tar;
```

Comment:
@ericstj @ViktorHofer @danmoseley  I added the assembly to this file because I didn't do it in main. This will ensure the new assembly gets shipped as part of the shared framework.

Reply:
Are there any validation steps (packaging build?) that you need to run locally with this change, because PR validation doesn't do it?

Reply:
I was told to submit this first to main. I'll force-push to the state before this commit.

Reply:
Why? We can't submit this until it's all done anyway, I suggest to leave it in the state you are now putting into main.

Reply:
If it goes in identically into main, then validation already ran here.

Reply:
Rewinded to the preview commit. Here is the PR for `main`: https://github.com/dotnet/runtime/pull/68343

Reply:
Apologies. GitHub didn't show your latest comments, Dan.